mne/decoding/time_gen.py

```diff
@@ -1375,3 +1468,23 @@ def _clean_times(self):
             self.y_pred_ = [y_pred[0] for y_pred in self.y_pred_]
         if hasattr(self, 'scores_'):
             self.scores_ = [score[0] for score in self.scores_]
+
+
+def _chunk_X(X, slices, gat, n_orig_epochs, test_epochs):
```

Comment:
function name should not have `capitals`. Call it `_chunk_data`. Naming matters @kingjr . It can make the difference between code that is readable like English text and dense code.